games-simulation/irrlamb/irrlamb-1.0.1r613.recipe

```diff
@@ -0,0 +1,64 @@
+
+SUMMARY="3D physics game"
+DESCRIPTION="irrlamb is a 3D game that probably involves a lot of physics \
+and frustrating gameplay."
+HOMEPAGE="https://github.com/jazztickets/irrlamb"
+COPYRIGHT="2015-2019 jazztickets"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/jazztickets/irrlamb/releases/download/v1.0.1/irrlamb-1.0.1r613-src.tar.gz"
+CHECKSUM_SHA256="8bb7d49a41c57170f4104c64176f975986b99646752f29749a4a1dabd702750a"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	irrlamb$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
+	lib:libglu$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libx11$secondaryArchSuffix
```

Comment:
The X11 port cannot draw. You can either port the game to use Haiku gui or abandon this.